### DISCUSSION TITLE
Grafana 7.3, 7.5 - Table Column Filters should not reset

### DISCUSSION THREAD

**Discussion by ashishkaransingh** [grafana/grafana#38272] [Dashboards, Panels & Visualizations]
**What would you like to be added**: Table column filter should remain set unless user change them or until user reload dashboard

![image](https://user-images.githubusercontent.com/29729110/116997864-fe1c4880-acfa-11eb-874c-6ae889004244.png)
 

![image](https://user-images.githubusercontent.com/29729110/116997483-79312f00-acfa-11eb-9efc-797eb4add8ab.png)

**Why is this needed**: We have hundreds of unique tags and it would be more useful if user set a filter and it remains, even on changing a datetime range or refresh panels etc. This would avoid setting the filter again and again.

Thanks

**Comment by dkroboth-oneweb** [comment]
Yes, would like this feature as well

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
